src/Curation/Curation.router.ts

```diff
@@ -5,35 +5,53 @@ import { HTTPError, STATUS_CODES } from '../common/HTTPError'
 import { withAuthentication, AuthRequest } from '../middleware'
 import { isCommitteeMember } from '../Committee'
 import { collectionAPI } from '../ethereum/api/collection'
+import { thirdPartyAPI } from '../ethereum/api/thirdParty'
 import { getValidator } from '../utils/validator'
-import { Collection } from '../Collection'
+import { Collection, CollectionService } from '../Collection'
+import { getMergedCollection, isTPCollection } from '../Collection/utils'
 import { Item } from '../Item'
-import { CurationStatus, patchCurationSchema } from './Curation.types'
+import { ItemCuration, ItemCurationAttributes } from './ItemCuration'
+import {
+  CurationStatus,
+  CurationType,
+  patchCurationSchema,
+} from './Curation.types'
 import { CurationService } from './Curation.service'
 import {
   NonExistentCollectionError,
   UnpublishedCollectionError,
 } from '../Collection/Collection.errors'
-import { CurationType } from '.'
-import { getMergedCollection } from '../Collection/utils'
 import {
   CollectionCuration,
   CollectionCurationAttributes,
 } from './CollectionCuration'
-import { ItemCurationAttributes } from './ItemCuration'
 
 const validator = getValidator()
 
 // TODO: Use CurationStatus everywhere
 export class CurationRouter extends Router {
   mount() {
+    // TODO: we might need to rename all endpoints to their actual entities:
+    //   - /collections/:id/curation -> /collectionCurations/:id
+    //   - /items/:id/curation -> /itemCurations/:id
```

Comment:
I think we have those, right?
- `'/collections/:id/curation'`
- `'/items/:id/curation'`

Reply:
I think that I misunderstood the comment, the comment was correct.
Instead of being `/collections/:id/curation`, as curation is not a resource, it should be curations if it was, the endpoint should be `/collectionCurations/:collectionId` If I'm not mistaken.